draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -304,38 +304,77 @@ preregister their mDNS names to speed up ICE gathering.
+                       |                |                 | <Resolve mDNS
+                       |                |                 |  name N1.1 to
+                       |                |                 |  192.168.1.1>
+                       |                |                 | <Resolve mDNS
+                       |                |                 |  name N1.2 to 
+                       |                |                 |  2001:db8::1> 
+                       |                |                 |
+      - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+                  ICE Agent 1 sends server-reflexive candidates
+                       |                |                 |
+        <192.168.1.1   |--Binding Req-->|                 | 
+         is 192.0.2.1> |<-Binding Resp--|                 | 
+           <192.0.2.1> |------ srflx Candidate C1.3 ----->| 
+      <2001:db8::1     |--Binding Req-->|                 | 
+       is 2001:db8::1> |<-Binding Resp--|                 | 
+         <2001:db8::1> |------ srflx Candidate C1.4 ----->| <Discard C1.4
+                       |                |                 |  as redundant>
+                       |                |                 |
+      - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+              ICE Agent 2 sends mDNS candidates, resolution is slow
+                       |                |                 |
+                       |<------ mDNS Candidate C2.1 ------| <N2.1>
+                       |<------ mDNS Candidate C2.2 ------| <N2.2>
+       <Resolve mDNS   |                |                 |     
+        name N2.1 ...> |                |                 |
+       <Resolve mDNS   |                |                 |     
+        name N2.2 ...> |                |                 |
+                       |                |                 |
+      - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+        ICE Agent 2 sends server-reflexive candidates, resolution completes
+                       |                |                 |
+                       |                |<--Binding Req---| <192.168.1.2 
+                       |                |---Binding Resp->|  is 192.0.2.2>
+                       |<----- srflx Candidate C2.3 ------| <192.0.2.2>
+                       |                |<--Binding Req---| <2001:db8::2
+                       |                |---Binding Resp->|  is 2001:db8::2>
+                       |<----- srflx Candidate C2.4 ------| <2001:db8::2>
+                       |                |                 |
+        <... N2.1 is   |                |                 |     
+         192.168.1.2>  |                |                 |
+        <... N2.2 is   |                |                 |     
+         2001:db8::2,  |                |                 |
+         discard C2.4> |                |                 |
```

Comment:
nit: maybe add "as redundant" as on line 337-338

Reply:
I considered it but omitted it since it was already mentioned for C1.4, but willing to be convinced.

Reply:
I agree it does read unnecessary after C1.4; just in case symmetry is preferred. The head commit LGTM.